include/bluetooth/mesh/health_faults.h

```diff
@@ -0,0 +1,85 @@
+/** @file
+ * @brief Bluetooth Mesh Health faults
+ */
+
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_HEALTH_FAULTS_H__
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_HEALTH_FAULTS_H__
+
+/**
+ * @brief List of specification defined Health Fault values.
+ * @defgroup bt_mesh_health_faults Bluetooth Mesh Health Faults
+ * @ingroup bt_mesh
+ * @{
+ */
+
+/** No fault has occurred. */
+#define BT_MESH_HEALTH_FAULT_NO_FAULT                           (0x00)
```

Comment:
We don't generally put parenthesis around atomic values (there's no way evaluation order could be affected no matter where the value is placed without parenthesis). So I'd drop it both because it's redundant as well as consistency with the rest of the stack.

Reply:
Makes sense, I'll change it!

Reply:
Fixed.